metadata_tools/hosts/GO_0xxx/gcp_geometry_config.yml

```
@@ -0,0 +1,13 @@
+provider: gcp
+gcp:
+  job_id:                   metadata_geometry-job
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)